posts/setting-up-a-shared-library-and-seed-job-in-jenkins-1.md

```diff
@@ -0,0 +1,134 @@
+## Introduction
+I have been working at a client the past few months that has adopted a desire for a consistent [JHipster](https://www.jhipster.tech/) microservice architecture. We have more than ten microservices that are supported by multiple teams. Developers support multiple services and we needed a uniform approach to deploy quality code quicly to the cloud. We set up Jenkins to support building and testing any branch of all services while maintaing the ability to deploy releases along side feature development. As Jenkins grew, we decided to maintain our Jenkins related code through source control. 
```

Comment:
'quicly'

Reply:
The latest version has this and other type-o's corrected